diem-unified-charter.md

```diff
@@ -1,49 +1,93 @@
+Digital emblems extend the range of identifying marks from the physical realm (visual and tactile) to the digital realm.
+The presence of a digital emblem represents a new signal available to cyber operators, one which can be combined with existing operational procedures to reduce uncertainty and improve decision making.
+
+The DIEM working group produces documents which describe the creation, presentation, validation, and interpretation of digital emblems.
+
+## Initial Scope
+
+Under the current charter, the working group is limited to the presentation / discovery of digital emblems for the following digital assets: 
+
+* domain names
+* hostnames
+* ip addresses
+
+Designs for QRCodes, NFC, Bluetooth, and other discovery mechanisms are currently out of scope.
+
+The working group will first describe a high level informational architecture and data model.
+After the architecture and data model are published as an informational RFC, the working group will specify a single concrete serialization and a discovery mechanism based on the DNS.
+The working group will coordinate with relevant working groups within the IETF and other SDOs to ensure that existing standards are leveraged appropriately.
+The working group will not produce any standard track generic serialization formats.
+The working group will not produce any standard track extensions to the DNS. 
+The working group will not develop novel security mechanisms, cryptographic primitives, or digital signature schemes.
+Until the architecture document is published, the working group will develop and maintain an informational use cases and requirements document, but is not required to request publication of this document.
+
+### Investigation of Serialization Formats
+
+The working group will research, analyze and evaluate application layer serialization formats for suitability to the initial scope.
+The working group will attempt to match the conventions for serializing data structures used in the discovery protocols, for example, key value strings in DNS records or structured field values in http.
+This work must be completed in the use cases and requirements document, and the architecture must not assume any specific serialization formats or discovery mechanisms.
+
+### Investigation of Securing Mechanisms
+
+The working group will research, analyze and evaluate securing mechanisms for digital emblems.
+The group is not required to describe object level securing mechanisms, in the case that existing protocol level securing mechanisms are sufficient, for example DNSSEC for DNS records, or TLS for well known HTTPS URLs.
+This work must be completed in the use cases and requirements document, and the architecture must not assume any specific securing mechanism or discovery mechanisms.
+
+### Design of Emblem Types
+
+The working group will describe at least one digital emblem suitable for indicating protection under international humanitarian law.
```

Comment:
```suggestion
The working group will describe at least one digital emblem suitable for indicating protection under IHL.
```

Now defined as an acronym in my above suggestion. Happy to keep it spelled out, too. Would then not introduce the acronym at all and expand it everywhere.